source/snippets/addRowUpdateRow.md.erb

```diff
@@ -1 +1 @@
-<li>One or more [*location-specifier* attributes](#row-location) (optional)</li><li>**expanded** (optional)</li><li>**format** (optional)</li><li>**cells** (optional) -- if specified, must be an array of [Cell objects](#cell-object), where each object is limited to the following attributes:</li><ul><li>columnId (required)</li><li>One of the following (required):</li><ul><li>value</li><li>formula</li></ul><li>When **value** is specified</li><ul><li>strict (optional)</li><li>[hyperlink](#hyperlink-object) (optional) with exactly one of the following attributes set:</li><ul><li>url</li><li>sheetId</li><li>reportId</li></ul><li>[linkInFromCell](#celllink-object) (optional) with all of the following attributes set:</li><ul><li>sheetId</li><li>rowId</li><li>columnId</li></ul></ul><li>overrideValidation (optional)</li><li>format (optional)</li></ul><li>**locked** (optional) - **true** to lock the row or **false** to unlock the row.</li></ul>See [Column Types](#column-types) for more information.<BR/><BR/>Please note the following:<ul><li>Column Ids must be valid for the sheet to which the row belongs, and must only be used once for each row in the operation.</li><li>Cells of a project sheet in the "Finish Date" column cannot be updated via API.</li><li>Cells of a project sheet in the  "Start Date" column cannot be updated via API for rows that contain a value in the "Predecessor" column.</li><li>Max length for a cell value is 4000 characters after which truncation occurs without warning. Empty string values are converted to null.</li><li>Calculation errors or problems with a formula will not cause the API call to return an error code. Instead, the response will contain the same value as in the UI, such as cell.value = "#CIRCULAR REFERENCE".</li> 
```

Comment:
Why removing the Start Date note?

Reply:
This relates to APIDOC-1068. I interpreted that work item as meaning that the Predecessor flag no longer blocks you from modifying the start date. There is still a restriction on start date for a parent row, but that is called out in a note in the text.

Reply:
I put the Start Date note back in based on email with Dave.